components/script/dom/htmlselectelement.rs

```diff
@@ -258,6 +258,36 @@ impl HTMLSelectElementMethods for HTMLSelectElement {
     fn Remove(&self) {
         self.upcast::<Element>().Remove()
     }
+
+    // https://html.spec.whatwg.org/multipage/#dom-select-value
+    fn Value(&self) -> DOMString {
+        self.upcast::<Node>()
+            .traverse_preorder()
```

Comment:
Same algorithm is done elsewhere in the file and was going to take care of it with https://github.com/servo/servo/issues/13763